src/components/DatePicker/docs/DatePickerWiringExample.tsx

```diff
@@ -11,11 +11,45 @@ export const DatePickerWiringExample = class DatePickerExample extends React.Com
   render() {
     const { value } = this.state;
     return (
-      <DatePicker
-        className={classes.root}
-        value={value}
-        onChange={this._onChange}
-      />
+      <>
+        <DatePicker
+          className={classes.root}
+          value={value}
+          onChange={this._onChange}
+        />
+        <br />
+        <br />
+        <h3>Calendar With Indications:</h3>
+        <br />
```

Comment:
I would try to avoid these `<br />` 😛

A wrapper with padding and margin-bottom might do the work

Reply:
You are right.. I will change it :)